crates/ironrdp-session/src/fast_path.rs

```diff
@@ -9,6 +9,8 @@ use ironrdp_pdu::codecs::rfx::FrameAcknowledgePdu;
 use ironrdp_pdu::fast_path::{FastPathHeader, FastPathUpdate, FastPathUpdatePdu, Fragmentation};
 use ironrdp_pdu::geometry::{InclusiveRectangle, Rectangle as _};
 use ironrdp_pdu::pointer::PointerUpdateData;
+#[cfg(feature = "qoi")]
+use ironrdp_pdu::rdp::capability_sets::CODEC_ID_QOI;
```

Comment:
suggestion: I would recommend to fully qualify at the use site, just so we don’t have cfgs in the imports.